script.js

```diff
@@ -42,21 +42,6 @@ document.addEventListener('DOMContentLoaded', function() {
     returnFocusToEl && returnFocusToEl.focus && returnFocusToEl.focus();
   }
 
-  // show form controls when the textarea receives focus or backbutton is used and value exists
-  var commentContainerTextarea = document.querySelector('.comment-container textarea'),
-    commentContainerFormControls = document.querySelector('.comment-form-controls, .comment-ccs');
-
-  if (commentContainerTextarea) {
-    commentContainerTextarea.addEventListener('focus', function focusCommentContainerTextarea() {
-      commentContainerFormControls.style.display = 'block';
```

Comment:
Removing this code seems to prevent the comment button from appearing

![Screenshot 2022-02-02 at 10 53 45](https://user-
images.githubusercontent.com/5488576/152131476-b739b2df-c4f0-4e03-86ec-384067fead6f.png)